recipes/shx

```diff
@@ -1,2 +1 @@
-(shx :repo "riscy/shx-for-emacs"
-     :fetcher github)
+(shx :repo "riscy/shx-for-emacs" :fetcher github)
```

Comment:
🤣 can't the CI be triggered manually? I suppose that it won't run anything though 😭

Reply:
Yeah, I'll drop this commit before merging. 😛  Ideally we have a suite of test recipes that run when no recipes change...